Warnings in code

```
pio_darray_int.c: In function ‘write_darray_multi_par’:
pio_darray_int.c:374:9: warning: unused variable ‘dsize’ [-Wunused-variable]
  374 |     int dsize;             /* Data size (for one region). */
      |         ^~~~~
pio_darray_int.c: In function ‘pio_read_darray_nc’:
pio_darray_int.c:1177:21: warning: unused variable ‘countlist’ [-Wunused-variable]
 1177 |         PIO_Offset *countlist[iodesc->maxregions];
      |                     ^~~~~~~~~
pio_darray_int.c:1176:21: warning: unused variable ‘startlist’ [-Wunused-variable]
 1176 |         PIO_Offset *startlist[iodesc->maxregions];
      |                     ^~~~~~~~~
pio_darray_int.c:1175:13: warning: unused variable ‘rrlen’ [-Wunused-variable]
 1175 |         int rrlen = 0;
      |             ^~~~~
pio_darray_int.c:1173:16: warning: variable ‘tmp_bufsize’ set but not used [-Wunused-but-set-variable]
 1173 |         size_t tmp_bufsize = 1;
      |                ^~~~~~~~~~~
pio_darray_int.c: In function ‘flush_output_buffer’:
pio_darray_int.c:1702:9: warning: unused variable ‘mpierr’ [-Wunused-variable]
 1702 |     int mpierr;  /* Return code from MPI functions. */
      |         ^~~~~~
```

--- a/src/clib/pio_darray_int.c
+++ b/src/clib/pio_darray_int.c
@@ -371,7 +371,9 @@ write_darray_multi_par(file_desc_t *file, int nvars, int fndims, const int *vari
 {
     iosystem_desc_t *ios;  /* Pointer to io system information. */
     var_desc_t *vdesc;    /* Pointer to var info struct. */
+#ifdef _PNETCDF
     int dsize;             /* Data size (for one region). */
+#endif
     int ierr = PIO_NOERR;
 #if USE_VARD_WRITE
     PIO_Offset gdim0;  /* global size of first dimension if no unlimited dimension and ndims<fndims */
@@ -1170,12 +1172,14 @@ pio_read_darray_nc(file_desc_t *file, io_desc_t *iodesc, int vid, void *iobuf)
         io_region *region;
         size_t start[fndims];
         size_t count[fndims];
-        size_t tmp_bufsize = 1;
         void *bufptr;
+#ifdef _PNETCDF
+        size_t tmp_bufsize = 1;
         int rrlen = 0;
         PIO_Offset *startlist[iodesc->maxregions];
         PIO_Offset *countlist[iodesc->maxregions];
-
+#endif
+	
         /* buffer is incremented by byte and loffset is in terms of
            the iodessc->mpitype so we need to multiply by the size of
            the mpitype. */
@@ -1194,7 +1198,6 @@ pio_read_darray_nc(file_desc_t *file, io_desc_t *iodesc, int vid, void *iobuf)
         /* For each regions, read the data. */
         for (int regioncnt = 0; regioncnt < iodesc->maxregions; regioncnt++)
         {
-            tmp_bufsize = 1;
             if (region == NULL || iodesc->llen == 0)
             {
                 /* No data for this region. */
@@ -1699,12 +1702,12 @@ pio_read_darray_nc_serial(file_desc_t *file, io_desc_t *iodesc, int vid,
 int
 flush_output_buffer(file_desc_t *file, bool force, PIO_Offset addsize)
 {
-    int mpierr;  /* Return code from MPI functions. */
     int ierr = PIO_NOERR;
 
 #ifdef _PNETCDF
     var_desc_t *vdesc;
     PIO_Offset usage = 0;
+    int mpierr;  /* Return code from MPI functions. */
 
     /* Check inputs. */
     pioassert(file, "invalid input", __FILE__, __LINE__);

--- a/tests/fncint/ftst_pio_orig.f90
+++ b/tests/fncint/ftst_pio_orig.f90
@@ -65,7 +65,7 @@ program ftst_pio
   call PIO_initdecomp(ioSystem, PIO_int, dims, compdof, iodesc)
 
   ! Create a file.
-  ierr = PIO_createfile(ioSystem, pioFileDesc, PIO_IOTYPE_PNETCDF, FILE_NAME, PIO_clobber)
+  ierr = PIO_createfile(ioSystem, pioFileDesc, PIO_IOTYPE_NETCDF, FILE_NAME, PIO_clobber)
   if (ierr .ne. nf_noerr) call handle_err(ierr)
 
   ! Define dimensions.

